### PR TITLE
address ethereum / javascript issues

### PR DESCRIPTION
**Pull request by faddat**
@blasrodri yeah, it's basically just bumping deps.  

Whole chain of things that happened when I added the security scanning tools in github.

Could rightly think of it as the results of #216 -- on the javascript front.

### CODE REVIEW

**Comment by blasrodri** [comment]
@faddat what was the issue? Or is it just bumping deps?

File: .github/workflows/rust-clippy.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+      - name: Run rust-clippy
+        run:
+          cargo clippy
+          --all-features
+          --message-format=json | clippy-sarif | tee rust-clippy-results.sarif | sarif-fmt
+        continue-on-error: true
+
+      - name: Upload analysis results to GitHub
+        uses: github/codeql-action/upload-sarif@v1
```
Comment (blasrodri) [inline .github/workflows/rust-clippy.yml:52]: wat dis? lol sarif

Comment (blasrodri) [reply .github/workflows/rust-clippy.yml:52]: security tool from Github

Comment (faddat) [reply .github/workflows/rust-clippy.yml:52]: yep

**Comment by faddat** [comment]
Followed up in slack

**Pull request closed without merging**